src/functionalTest/resources/features/F-004 - Post recordSittingRecords endpoint/F-004.feature

```diff
@@ -298,7 +330,15 @@ Feature: F-004 - Scenarios for the POST /recordSittingRecords endpoint
     Then a "negative" response is received with a "400 Bad Request" status code
     And the response contains "message" as "008 could not insert"
     And the response contains "errorRecords[0].errorCode" as "POTENTIAL_DUPLICATE_RECORD"
+    And the response contains "errorRecords[0].am" as "true"
```

Comment:
Could we check createdByName field as well

Reply:
solved in[6bf0f21](https://github.com/hmcts/jps-judicial-payment-service/pull/85/commits/6bf0f210e5a9b1293b0d45cc13e0cecfae860748)